nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -662,15 +681,4 @@ private List<String> filterByExtension(String tablePath, String extension) {
         .filter(f -> f.endsWith(extension))
         .collect(Collectors.toList());
   }
-
-  private static String addRecordsToFile(Table table, String filename) throws IOException {
```

Comment:
why the change to move this from here to the top?

Reply:
will review the changes and repush, maybe it was a thing of IntelliJ on reformatting.